File: app/src/java/java/com/longtoast/bilbil_api/dto/ReviewDTO.java
package com.longtoast.bilbil_api.dto;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class ReviewDTO {

    private Long reviewId;
    private Long transactionId;
    private Long reviewerId;
    private String reviewerNickname;
    private int rating;
    private String comment;
    private String createdAt;
}

File: app/src/main/java/com/longtoast/bilbil/dto/ReviewDTO.kt
package com.longtoast.bilbil.dto

data class ReviewDTO(
    val reviewId: Long,
    val transactionId: Long,
    val reviewerId: Long,
    val reviewerNickname: String?, // 리뷰 작성자 닉네임
    val rating: Int,
    val comment: String?,
    val createdAt: String?         // 작성일 (String)
)